cloud/migrate-mst-cloud/entire-database.md

```diff
@@ -0,0 +1,146 @@
+# Migrate entire database at once
+
+Migrate smaller databases by dumping and restoring the entire database at once.
+
+<highlight type="warning"> 
+Depending on your database size and  network speed,
+migration may take several hours. You can continue reading from your old
+database during this time, though performance may be slower. Writes made to your
+old database during migration might not be copied to Timescale Cloud. Plan your migration accordingly to avoid data loss. 
+</highlight>
+
+## Prerequisites
+
+- Check that the PostgreSQL utilities [`pg_dump`][pg_dump] and
+  [`pg_restore`][pg_restore] are installed:
+
+  ```bash
+  pg_dump --version
+  pg_restore --version
+  ```
```

Comment:
these are two separate commands. I'm not sure if that's the way we visualize it in our docs but would suggest to break it up

  ```bash
  pg_dump --version
  ```

  ```bash
  pg_restore --version
  ```

Reply:
same applies to Strategy 2